state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -434,17 +441,38 @@ impl DataSummaryPoller {
             let storage_summary = match result {
                 Ok(storage_summary) => storage_summary,
                 Err(error) => {
-                    trace!(
-                        "Error encountered when polling peer ({:?})! Error: {:?}",
-                        peer,
-                        error
+                    debug!(
+                        (LogSchema::new(LogEntry::StorageSummaryResponse)
+                            .event(LogEvent::PeerPollingError)
+                            .message(&format!(
+                                "Error encountered when polling peer {:?}. Error: {:?}",
+                                peer, error
+                            )))
                     );
                     continue;
                 }
             };
 
+            info!(
+                (LogSchema::new(LogEntry::StorageSummaryResponse)
+                    .event(LogEvent::PeerPollingSuccess)
+                    .message(&format!(
+                        "Successfully polled peer {:?}. Storage summary: {:?}",
+                        peer, storage_summary
+                    )))
+            );
+
             self.data_client.update_summary(peer, storage_summary);
             self.data_client.update_global_summary_cache();
+
+            info!(
+                (LogSchema::new(LogEntry::PeerStates)
+                    .event(LogEvent::AggregateSummary)
+                    .message(&format!(
+                        "Global data summary: {:?}",
+                        self.data_client.get_global_data_summary()
```

Comment:
Following from my comment above, I think it makes sense for us to log this (maybe `debug` level)? But I also have concerns about size. Specifically, if you look at the `GlobalDataSummary` object, it contains `synced_ledger_infos` in the `AdvertisedData`. When we `Debug` format those, they can be massive. To avoid this, we should override the `Debug` formatting for `GlobalDataSummary` ourselves so that we don't print those out entirely.  Feel free to do this in a follow up PR and we can sync :)

Reply:
I'm going to implement overriding `Debug` for `GlobalDataSummary` and other responses with potentially large payload for other storage request types in a follow-up PR.